foundry.toml

```
@@ -1,2 +1,3 @@
-[ci]
-fuzz-runs = 10_000
+[profile.ci]
+[fuzz]
+runs = 10_000
```

I think you are going for this, i.e. a CI profile with 10k runs. Current setup is an empty CI profile and `fuzz` is an alias for `profile.default.fuzz`
```suggestion
[profile.ci.fuzz]
runs = 10_000
```

cc @gakonst since I don't have permissions to merge anything here